allUpperCase.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+TEXT=$(cat $1)
+
+for WORD in $TEXT
+do
+    if [[ $WORD =~ [[:upper:]]+$ ]]; then
+        echo $WORD
+    fi
```

Comment:
```suggestion
   [[ $WORD =~ [0-9] ]] && continue

    SANITIZED_WORD=$(echo $WORD | tr -d '.')

    [[ $SANITIZED_WORD =~ [[:upper:]]+$ ]] && echo $SANITIZED_WORD
```
With this we can ignore "words" that contain numbers and then make sure that we remove unnecesary dots 🙂